src/train_classifier_g.py

```diff
@@ -6,16 +11,23 @@
 import os
 path = Path(os.path.abspath("."))
 
+""" connect to SQL 
+
+"""
 def train_model():
   s = Suite()
 
   print("Loading training data.")
-  train_dat = pd.read_csv(join(path, "src/data/training_data.csv"))
+  # executes a SQL query and returns a pd.DataFrame
+  # 3 columns: _id, text, label(0/1)
+  train_dat = pd.read_sql_query("SQL_QUERY_FOR_TRANING_DATA")
   s.set_train_set(train_dat)
-  test_dat = pd.read_csv(join(path, "src/data/test_data.csv"))
+  # 2 columns: _id, text
+  test_dat = pd.read_sql_query("SQL_QUERY_FOR_TEST_DATA")
```

Comment:
Two requests:
* Abstract this a bit. Add a new function, likely in another file, like retrieve_data(), which you call here. In that function, you can have your current read_csv call. I will then change that and only that file when running it with my data. This'll centralize my changes more and reduce the likelihood of a merge conflict.
* Choose the train/test split here for me. 70/30 is probably fine for now. I don't have any preconcieved notions of how I want my data split, and I see no difference in splitting it differently from how you do for the CSV.

Reply:
I think there's a misnomer here.. test_data isn't really used to test the performance of the model; it's the dataset you want to apply the model to. Train/test split and cross-validation is in suite.py.